Makefile

```diff
@@ -332,7 +332,7 @@ testing/rally/corpora:
 SMOKETEST_VERSIONS ?= latest
 # supported-os tests are exclude and hence they are not running as part of this process
 # since they are required to run against different versions in a different CI pipeline.
-SMOKETEST_DIRS = $$(find $(CURRENT_DIR)/testing/smoke -mindepth 1 -maxdepth 1 -type d | grep -v supported-os)
+SMOKETEST_DIRS = $$(find $(CURRENT_DIR)/testing/smoke -mindepth 1 -maxdepth 1 -type d | grep -v supported-os | jq -cnR '[inputs | select(length > 0)]')
```

Comment:
What's the reason for using the pipe with the jq?

Reply:
I allow us to return a JSON array instead of a list of paths separated by `\n`.

Reply:
This breaks the existing implementation in the Makefile:
https://github.com/elastic/apm-server/blob/ae7acf930a2ef7b6074ce1de847025a0fec76c0d/Makefile#L360-L367

if `smoketest/discover` is the one used in this particular changeset, and no other consumers maybe the `jq` pipe can be added to the `smoketest/discover` make goal instead?